tests/test_PluginLogs.py

```diff
@@ -208,7 +208,7 @@ def testLoggingPythonPlugin(self):
         pluginLogName = os.path.join(lsst.utils.getPackageDir('meas_base'), 'tests', 'plugin.log')
         directLog(log, pluginLogName)
         exposure, cat = self.dataset.realize(noise=0.0, schema=schema)
-        source = cat[0]
+        # source = cat[0]  # noqa F841: Not sure if this is deliberate
```

Comment:
Can we become sure, rather than just adding the note? If it doesn't break when you take it out, it looks very much like an oversight to me.